componentes/card/card.js

```diff
@@ -1,8 +1,16 @@
 import {servicos} from '../../js/servicos-list'
 import { formatter } from '../../js/formatmoney.js'
-
+import {getData} from '../../js/db-request.js'
 
 export const createMarkupCard = (servicesList = servicos) => {
+    const data = getData()
+    const printData = () => {
+        data.then((a) => {
+          console.log(a);
+        });
+      };
```

Comment:
voce pode usar async/await aqui tb

```suggestion
export const createMarkupCard = async (servicesList = servicos) => {
    const data = await getData()